website/content/docs/import/azurekv.mdx

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+page_title: Azure secret import source
+description: The Azure KeyVault source imports secrets from Azure to Vault.
+---
+
+# Azure secret import source
+
+Use the Azure source to import secret data from Azure KeyVault into your Vault instance.
+
+## Argument reference
+
+Refer to the [HCL syntax](/vault/docs/import#hcl-syntax-1) for arguments common to all source types.
+
+## Additional arguments
+
+Required:
+
+- `key_vault_uri` `(string)` - The URI of the Azure KeyVault to import from
```

Comment:
```suggestion
- `key_vault_uri` `(string)` - The URI of the Azure KeyVault you want to import from.
```
As written, it looks like the sentence stops suddenly. I'm providing a clarifying edit assuming that's not the case 😅